### PR TITLE
Contracted mobs wont recieve toxin damage by mistake anymore

### LINKED ISSUE #6770: Contractor kidnapping lethal to Oozelings :
## Round ID:
37790

## Reproduction:

Got kidnapped by a contractor as an oozeling - was fully fed and not hungry/injured/poisoned - after getting ransomed back, within seconds i started dying of massive toxin damage and the associated blood loss for oozelings. Passed out, was unable to be revived even by a full team of people trying surgery and defib, so whatever it was it REALLY fucked me up. 

Something about the contractor kidnap/ransom process seems like it's possibly trying to 'heal' toxin damage and as such killing the oozeling - the same thing happens if I ever take nanites for any reason as an oozeling - it tries to heal toxin damage, by giving me more toxin damage, until I die.

### PR DESCRIPTION
**Pull request by Salex08**
## About The Pull Request

Closes: https://github.com/BeeStation/BeeStation-Hornet/issues/6770

Once Contracted you would recieve ominizine which is harmful to oozelings and toxin lover mobs, this pull injects mobs a more friendly healing reagent instead

## Why It's Good For The Game

Mobs arnt supposed to be harmed when contracted

## Changelog
:cl:
fix: heals contracted mobs more properly
/:cl:

### CODE REVIEW

File: code/modules/antagonists/traitor/syndicate_contract.dm [hunk 1]
```diff
@@ -166,8 +166,11 @@
 	if (M.stat != DEAD)
 		// Heal them up - gets them out of crit/soft crit. If omnizine is removed in the future, this needs to be replaced with a
 		// method of healing them, consequence free, to a reasonable amount of health.
-		M.reagents.add_reagent(/datum/reagent/medicine/omnizine, 20)
-
+		//if the species is oozeling change the reagent to regen_ooze since omnizine deals toxin damage to them
+		if(is_species(M, /datum/species/oozeling))
```
Comment (PowerfulBacon) [inline code/modules/antagonists/traitor/syndicate_contract.dm]: This should handle all mobs that take inverse toxin damage (Toxin lover trait?)

Comment (PowerfulBacon) [reply code/modules/antagonists/traitor/syndicate_contract.dm]: (or you could just give regenerative nanites)

Comment (Salex08) [reply code/modules/antagonists/traitor/syndicate_contract.dm]: I mean yeah and if someone had omnizine on them before they could overdose

Comment (Salex08) [reply code/modules/antagonists/traitor/syndicate_contract.dm]: changed it alltogether, is it better now?

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**